### DISCUSSION TITLE
District without a website wants to add their meetings to our Area site & TSML

### DISCUSSION THREAD

**Discussion by fritzisa** [code4recovery/12-step-meeting-list#770] [Q&A] [answered]
[Area 59](https://area59aa.org), has a small district without a website, (our only one). They recently provided a list of their meetings with basic information, which has been saved as a CSV file (attached)
[District-50-Meeting-Listing-APR-15-2022.csv](https://github.com/code4recovery/12-step-meeting-list/files/8496896/District-50-Meeting-Listing-APR-15-2022.csv)
Is this sufficient information for TSML?

**Comment by kiyote33** [comment]
Hi @fritzisa, If you compare the District 50 listing with the example csv file (found on a tab in the Import & Settings page) you will see there might be some formatting issues.  I would recommend following the example file closely. Column names should be the same as in the example, although many column are optional and not required. Time and End Time formatting should be in an acceptable date string format (again, just use the format in the example). Address information should be as complete and precise as possible. 3rd & Brown may / may not resolve with Google. Zoom only is not an acceptable location. Zoom meetings don't need a physical address, but they should contain the City, State, and Country information of the sponsoring group. The Type field (should be Types) should follow the format and available options spelled out in the Spec document which you will see on the same page where the above noted example Template.csv file is found.
If you have a test environment, I would suggest testing it out before going live.  If not, send it here like the one you just provided and I'll be happy to test it for you.

Hope this helps.

**Reply by fritzisa** [reply]
Hi @kiyote33, Thanks so much for your direction. I cleaned up the addresses they provided and used the template you suggested. Would you please test it for me? Kindly let me know if I've overlooked anything, or if any changes are needed. I really appreciate your help!
Thank you!
[Area-59-District-50-APR-15-22.csv](https://github.com/code4recovery/12-step-meeting-list/files/8498180/Area-59-District-50-APR-15-22.csv)

**Reply by kiyote33** [reply]
Hi again,
Just a couple things I noticed: 
- Only 1 record didn't load, giving this error: **No location information provided for Wednesday Beginner.** As I mentioned before, if you want an Online meeting to load you will need to put a non-precise address in your .csv (something like Huntingdon, PA, USA).
- On the types, the Spec advises that each meeting should have a comma-separated list of types (i.e. Closed, Discussion, Wheelchair Access). My concern here is that the Meeting Guide app may not pick up your types because the comma delimiters are missing.
Other than that, things looked good...

**Comment by kiyote33** [comment, marked as answer]
[Area-59-District-50-APR-16-22 (c4r).csv](https://github.com/code4recovery/12-step-meeting-list/files/8500100/Area-59-District-50-APR-16-22.c4r.csv)
Hi @fritzisa, I updated your file with the conference_url and phone columns for online meetings as for some reason they are missing from the spec. I've also adjusted the types fields to valid types which will show up correctly in the types field. 
Hope this helps...

**Comment by fritzisa** [comment]
Hi @kiyote33, thanks so much for making those updates. I have imported the CSV into our [site](HTTP://area59aa.org) and their meetings have populated our meeting list of 3800+ meetings in Area 59. Can you please confirm that their meetings will populate the Meeting Guide app, or is there something more for me to do?
I really appreciate your help, this is our only district without a website, and we're thrilled to have been able to assist them in getting their meetings listed online.

**Reply by kiyote33** [reply]
Hi again,
Your district 50 meetings will filter through to the Meeting Guide App with the other 3800 records you have, assuming of course that GSO has been setup with the necessary configuration to retrieve the feed from your site.  That usually entails communicating with GSO and providing the URL which can be retrieved from the Feed Management section of the Settings tab. If your already getting your meetings on the Meeting Guide App then you should be good to go.

**Reply by fritzisa** [reply]
Hi @kiyote33,

I just got an email from my contact at District 50. [danifone54@gmail.com](url)

She said none of the meetings from Huntington, PA are showing up on the Meeting Guide app. I did a search and she's right. I looked for the meetings in Huntington on our Area website and found they were all listed.

https://area59aa.org/find-a-meeting/?region=huntingdon

This is the latest file (that you updated for me – see above), and I imported it.

[Area-59-District-50-APR-16-22.c4r.csv](https://github.com/code4recovery/12-step-meeting-list/files/8544127/Area-59-District-50-APR-16-22.c4r.csv)

Do you know what could be causing this issue?

Any help is greatly appreciated.

**Comment by fritzisa** [comment]
Excellent, our little rural district will be thrilled to have their meetings listed on our Area website and the "chair app," as they called it ;) Thanks again for your help!

**Comment by kiyote33** [comment]
Hi @fritzisa,

 

Are the other meetings on your website filtering though to the meeting guide app?

 

If they are then the Huntington meetings should be there unless they are being blocked by the Meeting Guide app itself. I don’t think they filter out meetings unless they are duplicates registered by someone else. In that case there should still be a meeting showing for the Huntington groups.

 

I would suggest that you contact GSO and ask their support staff to investigate why some of your meetings are not displaying. I believe that you can find more info on their website: Meeting Guide | Alcoholics Anonymous (aa.org) <https://www.aa.org/meeting-guide-app> .

 

Hope this helps

**Reply by kiyote33** [reply]
Hi @fritzisa, I got busy with other things and forgot to follow up on this. For that I apologize.  I looked at you area 59 site and I see you have 7 meetings listed for Huntingdon, PA. The feed for your site produces a JSON file which includes those same 7 meetings.  The meeting guide app appears to be listing other meetings that are available in your site meeting list, so it's very curious that they are not listing the Huntingdon meetings. Did you contact GSO and ask them about this? What was their response?

**Reply by ZamulaK** [reply]
Hi @kiyote33 - this is Dan B from Area 59, currently serving as Treasurer. Due to my programmer background, I've starting assisting with some of the Meeting Guide technical issues.

We are not directly feeding the Meeting Guide app from our website, instead we are just combining the feeds from various Intergroup and District sites throughout the Area. The other entities are the ones connected to the app and supplying the meeting updates. District 50 is currently not connected, but their meetings overlap geographically with our neighbor District 60, which was causing some confusion.

We are going to assist District 50 with getting connected to the app, and plan to help them set up a shared Google Sheet which can serve as the source feed. Thanks for all of your help!

**Reply by tech2serve** [reply]
HI @ZamulaK,

Hopefully this will get working for you. As you work through this, you might need to communicate with GSO to make sure geofencing (how they bound the geography included in a particular connected site) is set to accomplish your goals.

In general, I do recommend districts manage their own meetings lists so I'm glad to see y'all are going down this path. Generally, this results in more accurate meeting data.

However, feeds into Meeting Guide can be done individually or through a consolidator website (such as Area 59). Both approaches work, but we tend to see more duplicate meeting issues when *both* approaches are taken. Just something to keep in the back of your head.

And BTW, thank you for using your coding experiences to help them out! :-)

**Reply by ZamulaK** [reply]
Hi @tech2serve,

The pointers are appreciated! 

We only recently started aggregating the JSON feeds from the various sites throughout the Area to provide a consolidated view on the Area website. Many had connected to Meeting Guide long before that, so we don't plan to change any existing processes.

For District 50, we will make sure their connection request uses only the feed specific to them, and not the one from the Area. I agree it would confusing and problematic if *both* of us were sending the data to GSO.

Regarding the geofencing, we will certainly doublecheck, but I think it was just a misunderstanding on our part about the geographic boundaries at the western edge of our Area. If you check the [Test Google Sheet Feed](https://meetingguide.org/validate?url=https://docs.google.com/spreadsheets/d/1SsCWvD2-oZrKxNNDS3a1EKxD2bewzxV0TFU1PLXIYLk/edit?usp=sharing), there are only 16 meetings so it should be easy enough to verify!

**Comment by fritzisa** [comment]
Hi @kiyote33 <https://github.com/kiyote33 <https://github.com/kiyote33>> ,

I’m afraid GSO just sent me to the Meeting Guide helpdocs which I’ve already read through:

Dear Scott,
 
Warm greetings from the General Service Office of Alcoholics Anonymous U.S./Canada! My name is Mary, I am a past GSO staff member helping with the emails during the General Service Conference. It is good to have this chance to be in touch.
 
I am sorry to hear you are having problems with Meeting Guide listings. You can visit the Meeting Guide App Support Site for assistance: https://meetingguide.helpdocs.io/category/eebenj2xql-connecting-to-meeting-guide.
 
Sending best A.A. wishes and much gratitude for your service.
 
In fellowship,
 
Mary
 
Mary Cumings
Staff Correspondence for the
Northeast Regional Correspondent

I am a graphic designer, not an IT guy, so I am struggling with this issue.

I’m out of resources at this point.

Any help would be greatly appreciated.

In fellowship,

Scott Fritzinger  |  Area 59 – Web Servant  |  (610) 762-7792

**Comment by kiyote33** [comment]
Hi Scott,
Correct me if I'm wrong, but my understanding is that you are asking for assistance in determining why Huntington, PA meetings are not showing up on GSO's Meeting Guide App? The district that includes Huntington is registered with GSO, providing them with a feed link?
